configs/textdet/fcenet/fcenet_r50dcnv2_fpn_1200e_ctw1500.py

```diff
@@ -0,0 +1,136 @@
+fourier_degree = 5
```

Comment:
fourier_degree  is a parameter of model , head, or loss.

you'd better putting it in the model, head or loss